packages/client/lib/rpc/modules/engine.ts

```diff
@@ -264,6 +276,8 @@ export class Engine {
     try {
       await findBlock(toBuffer(parentHash), this.validBlocks, this.chain)
     } catch (error: any) {
+      // TODO if we can't find the parent, return ACCEPTED when optimistic sync is supported
```

Comment:
I think :thinking:  ACCEPTED is to be provided when payload doesn't extend canonical chain and can't be fully validated (because of no parent linkage available with the client yet or may be something else too), else SYNCING.